client/src/legacy/ArchiveAdmin/ArchiveAdmin.js

```diff
@@ -22,7 +22,7 @@ jQuery.entwine('ss', ($) => {
       const othersArchiveUrl = $(this).data('others-archive-url');
       if (targetClassName) {
         targetClassName = targetClassName.replace(/\\/g, '-');
-        const url = othersArchiveUrl + targetClassName;
+        const url = `${othersArchiveUrl}/${targetClassName}`;
```

Comment:
This assumes `othersArchiveUrl` will now _never_ have a trailing slash.... use `joinUrlPaths()` instead per https://docs.silverstripe.org/en/5/changelogs/5.0.0/#trailing-slash

Reply:
Updated